src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PreReleaseService.cs

```diff
@@ -51,13 +52,12 @@ public PreReleaseWindowStatus GetPreReleaseWindowStatus(Release release, DateTim
 
             var publishScheduled = release.PublishScheduled.Value;
             var startTime = GetStartTime(publishScheduled);
-            var endTime = GetEndTime(publishScheduled);
 
             return new PreReleaseWindowStatus
             {
-                Start = startTime,
-                End = endTime,
-                Access = GetAccess(release, startTime, endTime, referenceTime)
+                Start = GetStartTime(publishScheduled),
+                ScheduledPublishDate = publishScheduled,
```

Comment:
Is it worth keeping the variable names consistent and renaming this to `PublishScheduledDate`?